src/modules/common/TimeInput.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+import inputBase from '../../styles/inputs';
+
+const TimeInput = styled.input.attrs({
+  max: '20:00',
+  min: '16:00',
+  step: '900',
+  type: 'time',
+})`
```

Comment:
@csteinert please update the min/max/step to whatever makes sense. 4pm - 8pm with 15minute increments is what this gives you.